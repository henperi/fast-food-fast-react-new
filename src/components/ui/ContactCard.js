import React from "react";

export const ContactCard = () => (
  <div className="flex-item card" id="contact">
    <h2>Contact Us</h2>
    <form className="form-data container">
      <div className="form-member text-left">
        <label htmlFor="">Full Name</label>
        <input
          type="text"
          className="form-input"
          placeholder="Enter Fullname"
        />
      </div>
      <div className="form-member text-left">
        <label htmlFor="">Email</label>
        <input type="email" className="form-input" placeholder="Enter Email" />
      </div>
      <div className="form-member text-left">
        <label htmlFor="">Message</label>
        <textarea
          name=""
          id=""
          cols="8"
          rows="10"
          placeholder="Type a message"
        />
      </div>
      <button className="btn btn-blue btn-block btn-rounded btn-bg">
        Contact Us
      </button>
    </form>
  </div>
);
