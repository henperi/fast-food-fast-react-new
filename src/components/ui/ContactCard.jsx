import React from 'react';

const ContactCard = () => (
  <div className="flex-item card contact" id="contact">
    <h2>Contact Us</h2>
    <form className="form-data container">
      <div className="form-member text-left">
        <label htmlFor="fullname">
          Full Name
          <input type="text" id="fullname" className="form-input" placeholder="Enter Fullname" />
        </label>
      </div>
      <div className="form-member text-left">
        <label htmlFor="email">
          Email
          <input type="email" className="form-input" placeholder="Enter Email" />
        </label>
      </div>
      <div className="form-member text-left">
        <label htmlFor="messafe">
          Message
          <textarea name="" id="message" cols="8" rows="10" placeholder="Type a message" />
        </label>
      </div>
      <button type="button" className="btn btn-blue btn-block btn-rounded btn-bg">
        Contact Us
      </button>
    </form>
  </div>
);

export default ContactCard;
