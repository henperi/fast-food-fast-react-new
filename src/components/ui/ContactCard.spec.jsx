import React from 'react';
import { shallow } from 'enzyme';
import ContactCard from './ContactCard';

test('should render Header correctly', () => {
  const wrapper = shallow(<ContactCard />);
  expect(wrapper).toMatchSnapshot();
});
