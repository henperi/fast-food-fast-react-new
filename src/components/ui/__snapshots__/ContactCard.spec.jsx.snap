// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="flex-item card contact"
  id="contact"
>
  <h2>
    Contact Us
  </h2>
  <form
    className="form-data container"
  >
    <div
      className="form-member text-left"
    >
      <label
        htmlFor="fullname"
      >
        Full Name
        <input
          className="form-input"
          id="fullname"
          placeholder="Enter Fullname"
          type="text"
        />
      </label>
    </div>
    <div
      className="form-member text-left"
    >
      <label
        htmlFor="email"
      >
        Email
        <input
          className="form-input"
          placeholder="Enter Email"
          type="email"
        />
      </label>
    </div>
    <div
      className="form-member text-left"
    >
      <label
        htmlFor="messafe"
      >
        Message
        <textarea
          cols="8"
          id="message"
          name=""
          placeholder="Type a message"
          rows="10"
        />
      </label>
    </div>
    <button
      className="btn btn-blue btn-block btn-rounded btn-bg"
      type="button"
    >
      Contact Us
    </button>
  </form>
</div>
`;
